### PR TITLE
Suborbital Portal

### PR DESCRIPTION
**Pull request by clynnes**
Suborbital Portal (fka Airborne Portal) is ready to be unveiled after consultation with the Airborne Data Management Group in the IMPACT project.  It will see datasets with tagKey = 'gov.nasa.impact.* and includes a logo with hover version (lines brighten from grey to white).
(Note that the long "Sub-Orbital Catalog" name causes the name to appear in the line below the logos:  the ADMG prefer that to a version that shortens that name and appears next to the logos.)

### CODE REVIEW

**Comment by macrouch** [comment]
We're not a fan of the way the logo looks. We hadn't had a title as long as "Sub-Orbital Catalog" and it causes it to wrap under the logos
Screen Shot 2020-07-22 at 2 21 49 PM

Changing the `org` to "Sub-Orbital Catalog" and removing the `title` (defaults to Search) creates a much better looking logo area
Screen Shot 2020-07-22 at 2 32 10 PM

Would that be acceptable?

**Comment by clynnes** [comment]
> We're not a fan of the way the logo looks. We hadn't had a title as long as "Sub-Orbital Catalog" and it causes it to wrap under the logos
> Changing the `org` to "Sub-Orbital Catalog" and removing the `title` (defaults to Search) creates a much better looking logo area
> 
> Would that be acceptable?

I like your arrangement better (would have done it myself if I thought of the org trick), but I need to ask the customer.

**Comment by clynnes** [comment]
> > We're not a fan of the way the logo looks. We hadn't had a title as long as "Sub-Orbital Catalog" and it causes it to wrap under the logos
> > Changing the `org` to "Sub-Orbital Catalog" and removing the `title` (defaults to Search) creates a much better looking logo area
> > Would that be acceptable?

Customer loves it. Do you want me to withdraw the PR and submit a new one?  Or you guys want to make the change?

File: portals/suborbital/config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "hasStyles": true,
+  "hasScripts": false,
+  "hideCollectionFilters": true,
+  "logo": {
+    "id": "suborbital-logo",
+    "link": "https://earthdata.nasa.gov/esds/impact/admg/the-airborne-inventory",
+    "title": "Sub-Orbital Catalog"
+  },
+  "org": "NASA",
```
Comment (macrouch) [inline portals/suborbital/config.json:10]:
```suggestion
  "org": "Sub-Orbital Catalog",
```

File: portals/suborbital/config.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "pageTitle": "NASA Sub-Orbital Catalog",
+  "query": {
+    "tagKey": ["gov.nasa.impact.*"], 
+    "hasGranulesOrCwic": null
+  },
+  "title": "Sub-Orbital Catalog"
```
Comment (macrouch) [inline portals/suborbital/config.json:16]:
```suggestion
```

File: portals/suborbital/config.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "id": "suborbital-logo",
+    "link": "https://earthdata.nasa.gov/esds/impact/admg/the-airborne-inventory",
+    "title": "Sub-Orbital Catalog"
+  },
+  "org": "NASA",
+  "pageTitle": "NASA Sub-Orbital Catalog",
+  "query": {
+    "tagKey": ["gov.nasa.impact.*"], 
+    "hasGranulesOrCwic": null
+  },
```
Comment (macrouch) [inline portals/suborbital/config.json:15]:
```suggestion
  }
```

**Pull request merged**